clojure-mode.el

```diff
@@ -97,6 +97,21 @@
   "Face used to font-lock interop method names (camelCase)."
   :package-version '(clojure-mode . "3.0.0"))
 
+(defface clojure-global-constant-face
+  '((t (:inherit font-lock-constant-face)))
+  "Face used to font-lock the Clojure global constants: nil, true, false."
+  :package-version '(clojure-mode . "3.0.0"))
+
+(defface clojure-def-symbol-face
+  '((t (:inherit font-lock-keyword-face)))
+  "Face used to font-lock the symbols `def`, `defn`, and all detected variants (be them from clojure.core or not)."
+  :package-version '(clojure-mode . "3.0.0"))
+
+(defface clojure-lambda-arg-face
```

Comment:
I don't understand why they can't be font-locked as variables, as this is what they are. That seems like an odd decision to me. And I don't think we use the variable face for anything but them.